package.json

```diff
@@ -10,7 +10,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "@gen3/guppy": "0.1.2",
+    "@gen3/guppy": "git://github.com/uc-cdis/guppy.git#feat/dat",
```

Comment:
just to note down here in case we forget: change this after a new Guppy has been released